src/frontend/lib/intl.ts

```diff
@@ -44,7 +44,10 @@ function getUsableLanguages(
     if (!hasAtLeastOneTranslatedString) continue;
 
     if (!isSupportedLanguageTag(languageTag)) {
-      if (process.env.APP_VARIANT === 'development') {
+      if (
+        process.env.APP_VARIANT === 'development' &&
+        process.env.NODE_ENV !== 'test'
+      ) {
```

Comment:
got annoyed with this warning being logged when running the jest tests. This updates the logging condition such that it only logs when running the actual app (and not tests).